pom.xml

```diff
@@ -217,6 +217,12 @@
         <dep.wire.version>4.9.9</dep.wire.version>
         <dep.zookeeper.version>3.9.2</dep.zookeeper.version>
         <extraJavaVectorArgs>--add-modules=jdk.incubator.vector</extraJavaVectorArgs>
+        <!--
+        Instructs Provisio to write runtime dependencies in the format groupId_artifactId-version.jar.
```

Comment:
why is that not the default?

Reply:
It breaks mvnd which expects different file layout. Trino is indifferent to it